Solutions/VMWareESXi/Analytic Rules/ESXiDormantVMStarted.yaml

```diff
@@ -44,11 +44,14 @@ query: |
   | join (off_vms) on k
   | where p_off_vm !has DstHostname
   | summarize rec_p_on = makeset(DstHostname)

```

Comment:
make_set should be used and there should be a maximum limit provided.

Reply:
makeset limit added